Projekty/Projekt1/Jung_Kaczmarek_Marciniak/EDA_update.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<p>Ogólnie bardzo dobrze, jedyne do czego mógłbym się przyczepić to przydałoby się jakieś podsumowanie i informacje, jak to co zaobserwowaliście wpłynie na dalszą część projektu (inżynierię cech i budowę modelu). </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>